requirements.txt

```diff
@@ -8,9 +8,12 @@ pytz
 djangorestframework >= 3.10.3, < 3.12
 django-filter >= 2.2.0, < 2.3
 django-sekizai
+django-extensions >= 3.1.0
 inflect
 autobahn >= 17.9.3
 lunr == 0.5.6
+ipython >= 7.19.0
+jupyter == 1.0.0
```

Comment:
While I can see a point for having `ipython` in `requirements.txt`, the other two are too specialized. Move these to `requirements_extra.txt` instead - this is used for optional requirements like this.

Reply:
Alright. I will move those requirements to the extra file. 

So this works perfectly on my end, in fact I am using the notebooks to fine-tune my leveling algorithm. 

A couple of caveats and would need to be documented (not sure where) is 1) that you have to run ‘evennia shell_plus —notebook’ in the root of your game folder.  

2) when you create a new notebook you must run

‘’’python
import evennia
evennia._init()
‘’’

Reply:
For now you can add your info as a new section below this one: https://github.com/evennia/evennia/blob/develop/docs/source/Coding/Coding-Introduction.md#explore-evennia-interactively . You don't have to set up/build the docs locally if you don't want to, just add the text to the file and I'll rebuild next time I update them.